### DISCUSSION TITLE
custom fruit summons designed by DJs / custom API

### DISCUSSION THREAD

**Discussion by mcfiredrill** [datafruits/datafruits#1370] [Ideas]
DJs should be able to design their own fruit summons and determine the fruit ticket amount.

A simple editor should allow them to upload images/gifs/videos ? 
They can choose from several default spinning animations.
Set the price to what they want.

We could also have an API for fruit summon triggers, allowing people to build custom OBS plugins etc to react to fruit summons.

**Comment by dzuk-mutant** [comment]
that sounds really cool!
